socorro/signature/rules.py

```diff
@@ -554,6 +557,10 @@ def action(self, crash_data, result):
             result.debug(self.name, note)
         if signature:
             result.set_signature(self.name, signature)
+        if crashing_thread is not None:
+            result.extra["crashing_thread_id"] = crashing_thread
+        if crashing_thread_name:
+            result.extra["crashing_thread_name"] = crashing_thread_name
```

Comment:
Same here--you're not adding the `crashing_thread_name` or `crashing_thread_id` to the signature, so we don't need to make this change.